ecomap/api/v1_0/models/comment.py

```diff
@@ -1,10 +1,20 @@
-from sqlalchemy import Column, Integer, Text
-
-from api.v1_0.models import Base
+from sqlalchemy import Column, Integer, Text, ForeignKey, DateTime, Enum
+from sqlalchemy.orm import relationship
+from api.v1_0.models import Base, enum_activity_type
 
 
 class Comment(Base):
     __tablename__ = 'comments'
 
```

Comment:
as I remember, we discussed the idea to have modified_datetime and modified_user_id. And kill the activities at all.

This particular model is unacceptable because comment and changed comment will have different id's (and no connection at all).